gk/fib.c

```diff
@@ -1100,7 +1100,11 @@ del_fib_entry_locked(struct ip_prefix *ip_prefix, struct gk_config *gk_conf)
 	case GK_FWD_NEIGHBOR_FRONT_NET:
 		/* FALLTHROUGH */
 	case GK_FWD_NEIGHBOR_BACK_NET:
-		/* FALLTHROUGH */
+		RTE_LOG(WARNING, GATEKEEPER,
+			"gk: %s received a request to delete a FIB entry with prefix action %u; Gatekeeper may need to restart\n",
```

Comment:
Add prefix to the error message, this will be very helpful for an operator to understand the message.